doc/modules/clustering.rst

```diff
@@ -1370,7 +1370,7 @@ will not necessarily be close to zero.::
   - **Bounded range**: Lower values indicate different labelings, similar
     clusterings have a high (adjusted or unadjusted) Rand index, 1.0 is the
     perfect match score. The score range is [0, 1] for the unadjusted Rand index
-    and [-1, 1] for the adjusted Rand index.
+    and [-0.5, 1] for the adjusted Rand index.
```

Comment:
Sorry, but I didn't find the lower bound value for adjusted Rand index to be -0.5 in the attached new link. 

Can you provide on where in the research paper it's mentioned about, it incase I missed it ? 


But according to #8166 it's finalized that lower value should be -0.5, so I think it's fine to update the docs.

Reply:
Apparently it is https://arxiv.org/pdf/2002.03677.pdf from https://github.com/scikit-learn/scikit-learn/issues/8166#issuecomment-1196000573

Reply:
Exactly, in the paper they write "If in addition the clustering sizes are allowed to vary, then it is easily seen that the minimum possible value of the ARI is −1/2,". Sorry I only linked #8166 in the issue.